clients/mobile/src/setup/notification/saveDeviceToken.ts

```diff
@@ -1,10 +1,18 @@
 import { Platform } from 'react-native'
+import authService from '../../authentication/services/authService'
+import StorageService from '../../utility/storageService'
+import { TOKEN } from '../constant/token'
 
 export const saveDeviceTokenToSNS = async(
   deviceToken: string,
   fetchClient: any
 ): Promise<void> => {
   try {
+    if (await checkDeviceAlreadyRegisteredForSameUser()) {
+      console.log('Device already registered with the same user, skipping registration again')
```

Comment:
are you sure we're using console.logs? or should we use pino? the lambda stuff I belive uses pino

Reply:
This is for mobile application, `console.log` will be shown only on terminal for debugging and there is no other purpose of this in real Application.

Reply:
removed